src/tensor_ops.jl

```diff
@@ -256,6 +256,23 @@ end
 Base.ctranspose(S::AllTensors) = transpose(S)
 
 
+############################
+# Symmetric/Skew-symmetric #
+############################
+
+"""
+Computes the symmetric part of a second order tensor.
```

Comment:
you should at least say that this converts it to a `SymmetricTensor` if we keep this. It is after all possible for a `Tensor` to be symmetric.